chromium_src/chrome/browser/ui/webui/settings/site_settings_handler.cc

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/ui/webui/settings/site_settings_handler.h"
+#include "brave/components/brave_shields/browser/brave_shields_util.h"
+#include "chrome/grit/generated_resources.h"
+#include "components/grit/brave_components_strings.h"
+
+namespace {
+constexpr int kOriginalNotValidWebAddressId =
+    IDS_SETTINGS_NOT_VALID_WEB_ADDRESS_FOR_CONTENT_TYPE;
+}
+
+#undef IDS_SETTINGS_NOT_VALID_WEB_ADDRESS_FOR_CONTENT_TYPE
+
+#define IDS_SETTINGS_NOT_VALID_WEB_ADDRESS_FOR_CONTENT_TYPE                \
```

Comment:
`SiteSettingsHandler` -> `BraveSiteSettingsHandler` settings_ui.cc and then just register a new callback for `isPatternValidForType` that calls the superclass method and then does the additional check if it returns true

Reply:
done